### PR TITLE
Support linking binaries against prebuilt chainerx

### PR DESCRIPTION
**Pull request by shinh**

### CODE REVIEW

**Comment by shinh** [comment]
ChainerCI, test this please

File: scripts/run-travis-tests.sh [hunk 1]
```diff
@@ -17,7 +17,8 @@ cd build
 time cmake .. \
       -DCHAINER_COMPILER_ENABLE_PYTHON=ON \
       -DPYTHON_EXECUTABLE=/usr/bin/python3 \
-      -DCHAINER_COMPILER_ENABLE_OPENCV=ON
+      -DCHAINER_COMPILER_ENABLE_OPENCV=ON \
+      -DCHAINER_COMPILER_PREBUILT_CHAINERX_DIR=/usr/local/lib/python3.6/dist-packages/chainerx
```
Comment (take-cheeze) [inline scripts/run-travis-tests.sh]:
```suggestion
      -DCHAINER_COMPILER_PREBUILT_CHAINERX_DIR=$(pip3 show chainer | grep Location | sed -e 's/Location: //')/chainerx
```

Seems more general(sorry for dirty shell stuff).

**Comment by shinh** [comment]
ChainerCI, test this plea

File: scripts/run-travis-tests.sh (hunk 1 above)
Comment (shinh) [reply scripts/run-travis-tests.sh]: It worked nicely (I slightly golfed the shell script though), thanks!

**Pull request merged**